conda_smithy/lint_recipe.py

```diff
@@ -483,19 +483,18 @@ def run_conda_forge_specific(
 
     # 4: Do not delete example recipe
     if is_staged_recipes and recipe_dir is not None:
-        for recipe_name in ("meta.yaml", "recipe.yaml"):
-            example_fname = os.path.abspath(
-                os.path.join(recipe_dir, "..", "example", recipe_name)
-            )
```

Comment:
Why not this instead?

```python
for example_path in (("example", "meta.yaml"), ("example-v1", "recipe.yaml")):
    example_fname = os.path.abspath(os.path.join(recipe_dir, "..", *example_path))
```

And then adjust the f-string in the error msg to point to `os.path.join("recipes", *example_path)`?